site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -262,4 +262,4 @@
             </tbody>
         </table>
     </div>
-</div>
+</div>
```

Comment:
This whitespace-only change appears to be inadvertent and should be reverted.

Reply:
Fixed